src/safeds_stubgen/api_analyzer/_api.py

```diff
@@ -268,7 +270,7 @@ class Parameter:
     name: str
     is_optional: bool
     # We do not support default values that aren't core classes or classes definied in the package we analyze.
-    default_value: str | bool | int | float | None
+    default_value: str | bool | int | float | None | UnknownType
```

Comment:
While the syntax is the same (`unknown`) for an unknown value and an unknown type and the semantics only depend on the context, we should still be explicit here and create a class `UnknownValue`. The `default_value` is a value, not a type after all.